draft-ietf-ppm-dap.md

```diff
@@ -1523,39 +1519,34 @@ enum {
 } ExtensionType;
 ~~~
 
-Field "extension_type" indicates the type of extension, and "extension_data"
-contains information specific to the extension.
-
-Extensions are mandatory-to-implement: If an Aggregator receives a report
-containing an extension it does not recognize, then it MUST reject the report.
-(See {{input-share-validation}} for details.)
+Field `extension_type` indicates the type of extension, and `extension_data`
+contains the opaque encoding of the extension.
```

Comment:
Brevity: {{input-share-validation}} already has this MUST in it
Clarity: be exact about what `extension_data` is